data/scripts.js

```diff
@@ -897,7 +897,7 @@ exports.BattleScripts = {
 	},
 
 	runMegaEvo: function (pokemon) {
-		const isUltraBurst = !pokemon.canMegaEvo;
+		const effectType = pokemon.canMegaEvo ? '-mega' : '-burst';
```

Comment:
**I'D** prefer leaving it a boolean. Would have saved the recent huge bug and also would be more readable. Also `effectType` already means something else.